From a3c2f61649afd05e4bab457370f0b32d7644505c Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Mon, 3 Mar 2025 08:24:51 +0000
Subject: [PATCH] vpu_wrapper_hantro_encoder: fix missing include

gettimeofday is declared in sys/time.h. Add the include.

Fixes the build error:
| ../git/vpu_wrapper_hantro_encoder.c:953:3: error: implicit declaration of function 'gettimeofday' [-Wimplicit-function-declaration]
|   953 |   gettimeofday (&pObj->tvEnd, NULL);

Upstream-Status: Pending
Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 vpu_wrapper_hantro_encoder.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vpu_wrapper_hantro_encoder.c b/vpu_wrapper_hantro_encoder.c
index 3819c389320c..a0bdb2f4f6c0 100755
--- a/vpu_wrapper_hantro_encoder.c
+++ b/vpu_wrapper_hantro_encoder.c
@@ -25,6 +25,7 @@
 #include <string.h>
 #include <time.h>
 #include <semaphore.h>
+#include <sys/time.h>
 
 #include "headers/OMX_Video.h"
 #include "headers/OMX_VideoExt.h"
-- 
2.42.0

